rebar.lock

```diff
@@ -69,7 +69,7 @@
   0},
  {<<"helium_proto">>,
   {git,"https://github.com/helium/proto.git",
-       {ref,"ba0e3f6a35ec9397b8f0eb3bc832dbfaa95775a9"}},
```

Comment:
This was bumped by hand, we probably need to bump it through blockchain core properly.